docs/podman-container-runlabel.1.md

```diff
@@ -97,6 +98,10 @@ Require HTTPS and verify certificates when contacting registries (default: true)
 then TLS verification will be used. If set to false, then TLS verification will not be used. If not specified,
 TLS verification will be used unless the target registry is listed as an insecure registry in registries.conf
 
+**--replace**
+
+If a container exists of the default or given name, it will be replaced with a container from this image
```

Comment:
I think you should be more explicit here.  Replace means you will force delete the previous container (even if it is running) and then make a new one.